pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -140,30 +150,24 @@ func (c *cephStatusChecker) checkStatus() {
 }
 
 // updateStatus updates an object with a given status
-func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, condition cephv1.ConditionType, reason, message string) error {
-	ctx := context.TODO()
+func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, condition cephv1.ConditionType, reason, message string, conditionStatus v1.ConditionStatus) {
```

Comment:
Why don't we return an error anymore?

Reply:
The callers were only logging the error. Updating the status has been treated as best effort, so the main reconcile won't be blocked by updates to the status. Anyway, the status will be updated at the next 60s when the status is queried again, even if a single call fails.